POS/Operation/03_EditBillList.md

```diff
@@ -3,181 +3,172 @@
 
-Additional or alternative to the single discount, you can add a customized discount to the whole purchase in your bill list.
 
 #### Prerequisites
 
 - Offers for POS are created, see [Manage offers for POS](../Integration/07_ManageOffers.md).
-- At least one product is added to the bill list, see [Select an offer](./02_SelectOffer.md).
+- At least one offer is added to the bill list, see [Select an offer](./02_SelectOffer.md#select-an-offer).
 
 #### Procedure
 
 *Venduo POS > Sales > Select store and pay desk > Tab CASHPOINT*
 
 ![Cashpoint](../../Assets/Screenshots/POS/Sales/Cashpoint/Product.png "[Cashpoint]")
 
-1. Click the [PAY] button at the bottom of the bill list.   
-    The payment view is displayed in the bill list.
-
-    ![Pay](../../Assets/Screenshots/POS/Sales/Cashpoint/BillList/Pay.png "[Pay]")
-
-2. Click the [+ DISCOUNT] button above the total.   
-    The *Discount on bill* view is displayed.
+1. Click the offer in the bill list whose amount you want to change.   
+    The detail view of the offer is displayed in the bill list.
 
-    ![Discount on bill](../../Assets/Screenshots/POS/Sales/Cashpoint/BillList/DiscountBill.png "[Discount on bill]")
+    ![Offer detail](../../Assets/Screenshots/POS/Sales/Cashpoint/BillList/ProductDetails.png "[Offer detail]")
 
-3. Enter a description in the *Description* field to specify the discount, for instance campaign, defect.
-
-    > [Info] The discount to the purchase is always a percentage discount.
+2. To the right of the *Price* field, click the ![Edit](../../Assets/Icons/Edit02.png "[Edit]") (Edit) button.   
+    The price input field becomes editable. The field name changes to *Price (mod.)*.
 
-[comment]: <> (is that right? Why is there the percentage button but no other selection?)
+3.  Enter a price in the input field to the right of the *Price (mod.)* field.   
 
-4. Enter a percentage in the text field to the right of the *Description* field or select a percentage by clicking the number buttons in the lower area.   
-    The reduced price of the purchase is displayed below the percentage field.
+    > [Info] If you want to restore the original price, click the ![Reject](../../Assets/Icons/Cross03.png "[Reject]") (Reject) button in the *Price (mod.)* field.
 
-5. Click the [APPLY DISCOUNT] button in the upper right corner.   
-    The bill discount view is closed. The payment view is displayed again. The discount is indicated both as a percentage and as a fixed amount above the reduced total.
+4. Click the [Save] button in the bottom right corner to apply the price change.   
+    The detail view of the offer is closed. The bill list is displayed again. The modified price is displayed in italics to the right of the offer. <!---NEU-->The old price is listed one line below and crossed out. The price difference is stored for later analyzes.
```

Comment:
The price difference is stored for later **analyzes**. --> analysis?
was ist damit gemeint?

Reply:
man kann das auswerten